plugins/cd/__init__.py

```diff
@@ -25,40 +25,45 @@
 # from your version.
 
 import dbus
-from fcntl import ioctl
 import logging
-import os
-import struct
+import os.path
+import sys
+
+from gi.repository import GLib
 
 from xl.nls import gettext as _
-from xl import providers, event
+from xl import providers, event, main
 from xl.hal import Handler, UDisksProvider
 from xl.devices import Device, KeyedDevice
-
-from xl import playlist, trax, common
-import os.path
+from xl import playlist, trax, common, settings
+from xl.trax import Track
 
 import cdprefs
+import _cdguipanel
+from __builtin__ import staticmethod
```

Comment:
Hm?

Reply:
I'm removing this unused decorator import.